py/janitor/site/templates/index.html

```diff
@@ -1,8 +1,4 @@
-{% set layout = 'plain' %}
```

Comment:
What is this fixing?

Reply:
Rather than having to manual use `{% set layout = 'plain' %}` on places which doesn't want the sidebar, if the sidebar block is defined, it will alter the layout automaticlly. 
Otherwise, for single pages which don't have sidebars (such as `/` and `/about`) no sidebar will be the layout.

Summary: its one less thing to remember/worry about - and sidebar's div tags will automaticlly be sorted if there needs to be a sidebar.